README.md

```diff
@@ -1,6 +1,6 @@
 # Monte Carlo Data Collector - Snowflake Artemis Agent
 
-Monte Carlo's [SNA agent](https://hub.docker.com/r/montecarlodata/sna-agent).
+Monte Carlo's [SNA agent](https://docs.getmontecarlo.com/docs/sna-agent-deployment).
```

Comment:
Like the other agents, I think this can still be a link to the public DockerHub page. We already link the docs below.